AUTH.md

```diff
@@ -208,15 +205,15 @@ NEXT_PUBLIC_AZURE_AD_SCOPE_API=
 ## Hybrid Flow
 It is possible to utilize both OAuth2 and Basic authentication security schemes at the same time. With this configuration the end user will have to authenticate against an authorization server that supports `OAuth2 Authorization Code Flow with PKCE`.
```

Comment:
I think this might be a little imprecise, (I know it is not a part of the commit though), but when `OAuth2Enabled=true` we always use `Bearer` between the frontend and backend. As for the WITSML communication with backend, we always use `Basic`. Maybe we should rewrite this a little what do you think?

Reply:
Maybe the document should in general be split up into separate files for MsalDisabled, MsalEnabled, and Swagger, to make it clearer.

Reply:
Yes I like that idea, that will remove a lot of confusion I think 👍

Reply:
Should be a separate task I think

Reply:
Created subtask WE-784 to address this